README.adoc

```diff
@@ -57,6 +57,12 @@ See `Jvm.Main` in the `src/test` as an example.
 NOTE: If you specify system properties on the command line then they will override those in the
 `system.properties` file.
 
+== ChronicleInit
+
+The link:https://github.com/OpenHFT/Chronicle-Core/blob/ea/src/main/java/net/openhft/chronicle/core/ChronicleInit.java[`ChronicInit`] class handles application code which must be loaded/run before any system property is read and may override system properties.
```

Comment:
Typo? ChronicleInit (le missing)

Reply:
(reset approval)

Reply:
"handles application code" maybe say something like allows the developer to hook in their own code to be run at startup before the Chronicle static initialisers are run. These do things like load system properties